pandas/_libs/interval.pyx

```diff
@@ -281,6 +282,7 @@ cdef bint _interval_like(other):
             and hasattr(other, "closed"))
 
 
+@set_module("pandas")
 cdef class Interval(IntervalMixin):
```

Comment:
For Interval, it seems this is not working (see the build failures) because this is a `cdef class` instead of just a `class`. 

I don't directly know the best workaround, but so can also leave out Interval from this PR and already do the others

Reply:
OK will do.